model/DBHelper.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace SchemaZen.Library {
 	public class DBHelper {
-		public static bool EchoSql = false;
+        public static bool EchoSql = false;
```

Comment:
Are these spacing changes necessary, meaning were they off by one space, or was the white space an accident?  These are the types of things that are probably off-putting to the owner of the project.